balance/balance.go

```diff
@@ -0,0 +1,27 @@
+package balance
```

Comment:
Balance is something really generic, what do you think about keeping it in the root package?

Reply:
Moving this to the root package would create a circular dependency between `messagebird` and `messagebirdtest` (latter depends on `messagebird.Client`, and `balance_test.go` uses the `messagebirdtest` package for test server/client). Additionally, I guess this is a bit more consistent. Is it ok with you to keep it like this?